.circleci/config.yml

```diff
@@ -125,7 +125,8 @@ jobs:
           paths:
             # In Next.js 8 it will be
             - ./node_modules/.cache/next-minifier
-
+            - ./node_modules
```

Comment:
Why add `node_modules`? I don't think it's useful.

Should we remove `./node_modules/.cache/next-minifier`? Let's double check but I think it was actually replaced by `./.next/cache`.